pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllRunSamples.sql

```diff
@@ -3,7 +3,11 @@ SELECT part.partitionId
   ,bc1.sequence barcode
   ,bc2.sequence barcode_two
   ,tr.alias targeted_sequencing 
-FROM _Partition part 
+  ,(SELECT DISTINCT dataManglingPolicy FROM Platform JOIN SequencingContainerModel_Platform ON SequencingContainerModel_Platform.platformId = Platform.platformId WHERE SequencingContainerModel_Platform.sequencingContainerModelId = scm.sequencingContainerModelId) dataManglingPolicy
```

Comment:
I'm getting the following error: 
`nested exception is java.sql.SQLException: Subquery returns more than 1 row"`
when looking up run 3055, which has an "Unknown 4-Lane Illumina Flow Cell" container model.